xkcd_get.py

```diff
@@ -0,0 +1,58 @@
+import requests
+import json
+import wget
+import logging
+import os
+import numpy as np
+from PIL import Image
+
+
+    
+
+def xkcd_get_img():
+    logging.info("Downloading xkcd-json")
+    response = requests.get("https://xkcd.com/info.0.json")
+    result = response.json()
+
+    logging.info("Downloading xkcd_img")
+    logging.info(result["img"])
+
+    path = 'path/to/your/waveshare/implementation'
```

Comment:
You can get current directory using `os.path.dirname(os.path.realpath(__file__))`